htdocs/comm/propal/card.php

```diff
@@ -1562,7 +1562,7 @@
 		// Contacts (ask contact only if thirdparty already defined).
 		print "<tr><td>".$langs->trans("DefaultContact").'</td><td>';
 		print img_picto('', 'contact');
-		$form->select_contacts($soc->id, $contactid, 'contactid', 1, $srccontactslist);
+		$form->selectcontacts($soc->id, $contactid, 'contactid', 1, $srccontactslist);
```

Comment:
selectcontacts is not same than select_contacts
You must add a print select_contacts to get the same result.